All_materials/20230822_Web_Scraping_Financial_Ombudsman_Service/src/utils/file_paths.py

```diff
@@ -0,0 +1,20 @@
+"""
+Purpose of the script: loads config
+"""
+import logging
+import toml
```

Comment:
you need a requirements file that this needs to be imported

Reply:
done